src/command.ts

```diff
@@ -122,8 +122,9 @@ export class Command {
      * Spawn this with using node-pty
      * @param terminal Where the output stream will be sent
      * @param token For cancellation. This will send SIGINT on the process when cancelled.
+     * @returns the process
      */
-    spawn(terminal: OutputTerminal, token: CancellationToken): Promise<void> {
+    spawn(terminal: OutputTerminal, token: CancellationToken): Promise<pty.IPty> {
```

Comment:
I wonder if we should have a small interface to avoid making the pty.IPty as part of the "public api"

Reply:
I think it is not interruptive enough to have its own function, if that is what you mean by interface

Reply:
Oh I meant returning something like 
```typescript
class PseudoTerminal {
   private inner: pty.Ipty,

   dimensions() {
    return inner.dimensions
   }
}
```

Reply:
Ah, I see that will work, but not sure if it is worth it, since the pty is killed/exited when the `pty.IPty` was returned.